hooks/000-provide-uids-gids.chroot

```diff
@@ -124,6 +124,7 @@ colord:x:113:123:colord colour management daemon:/var/lib/colord:/usr/sbin/nolog
 gnome-initial-setup:x:114:65534::/run/gnome-initial-setup/:/bin/false
 rtkit:x:115:124:RealtimeKit,,,:/proc:/usr/sbin/nologin
 pipewire:x:116:125:Pipewire,,,:/proc:/usr/sbin/nologin
+polkitd:x:999:999:polkit:/nonexistent:/usr/sbin/nologin
```

Comment:
We've been following the existing practice of allocating sequential IDs for system users and groups. Is there any reason to put this up at 999?

Reply:
That wasn't intentional, I've fixed it